share/dictionary/dhcpv6/dictionary.rfc4704

```diff
@@ -12,9 +12,8 @@
 ##############################################################################
 
 ATTRIBUTE	Client-FQDN				39	struct
-MEMBER		Client-FQDN-Flags			uint8
+MEMBER		Client-FQDN-Flag-Reserved		bit[5]
+MEMBER		Client-FQDN-Flag-Update			bit[1]
+MEMBER		Client-FQDN-Flag-Override		bit[1]
+MEMBER		Client-FQDN-Flag-Forward		bit[1]
```

Comment:
I'm not sure why this is "forward".  It should be `Client-FQDN-Flag-Server-Update`